### DISCUSSION TITLE
Welcome to Project-Guidance Discussions!

### DISCUSSION THREAD

**Discussion by Kushal997-das** [Kushal997-das/Project-Guidance#12] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Kushal997-das** [comment, marked as answer]
Welcome :D

**Comment by TawfikYasser** [comment]
I'm very happy to be one of this great community, I'm Tawfik Yasser Computer Science at Cairo University, Egypt.

I have experience in Java, C++, SQL, MySQL, JSP, Servlets, Data structures and algorithms, Design patterns, SOLID, and git.

Currently I'm learning Advanced topics in data structures and algorithms, Also APIs.

I know HTML5, CSS, JavaScript, Ajax, Spring Boot, and Android.

I'm very happy to contribute to this community.

Thanks

**Reply by Kushal997-das** [reply]
Hello @TawfikYasser yes of course you can contribute.
